File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Customers/ICustomerData.php
<?php

namespace Ohtap\ECommerce\DomainModels\Objects\Customers;


use  Ohtap\ECommerce\DomainModels\Core\Object as  CoreObjects ;




interface ICustomerData extends  ICustomerDateOfBirth,ICustomerNationality,ICustomerPhoto,ICustomerCitizenNumber,ICustomerEmail,ICustomerGender,ICustomerPhone,CoreObjects\IObjectID, CoreObjects\IObjectCode, CoreObjects\IObjectDescription, CoreObjects\IObjectName, CoreObjects\IObjectOrder, CoreObjects\IObjectState, CoreObjects\IObjectTitle
{

}

File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Models Objects/Customers/ICustomerData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CoreObject = Ohtap.ECommerce.Domain.Models.Core.Object;

namespace Ohtap.ECommerce.Domain.Models.Objects.Customers
{
    public interface ICustomerData :  CoreObject.IObjectID, CoreObject.IObjectCode, CoreObject.IObjectName, CoreObject.IObjectOrder, CoreObject.IObjectDescription, CoreObject.IObjectState, ICustomerCitizenNumber, ICustomerGender, ICustomerPhoto, ICustomerPhone, ICustomerEMail, ICustomerDateOfBirth, ICustomerNationality
    {

    }
}
